text/0808-library-quality-scoring.md

```diff
@@ -0,0 +1,713 @@
+# RFC: Feature Flag Advisor
+
+* Original Author(s): @huang-benny
+* Tracking Issue: [#808](https://github.com/aws/aws-cdk-rfcs/issues/808)
+* API Bar Raiser: @kaizencc
+
+CDK developers and AWS customers often struggle to know whether a library is actively maintained, well-tested, and safe to adopt.
+Today, this requires manually checking repos, scanning docs, and guessing at signals like stars or commit history. Library Quality
+Scoring replaces that guesswork with a clear, explainable score backed by evidence, so customers can differentiate libraries at a
+glance and maintainers know exactly how to improve.
+
+## Working Backwards
+
+### README
+
+#### CDK Construct Analyzer
+
+`@cdklabs/cdk-construct-analyzer` is a CLI and library for evaluating the quality of construct libraries.
+
+It calculates a single score (0–100) based on three aspects:
+
+* **Maintenance**: Is the project actively maintained and are owners/maintainers responsive?
+* **Quality**: Does the project have good docs, tests, linting, and hygiene?
+* **Popularity**: How widely is the library adopted in the community?
+
+> [!WARNING]  
+> **Important Usage Guidelines**: This tool provides automated scoring based on publicly available metrics
+and can be used as **one factor** in your evaluation process, not as the sole decision making criteria.
+Scores reflect measurable signals but cannot capture all aspects of library quality, such as code
+architecture, security practices, or alignment with your specific use case. Always combine these scores with
+your own technical evaluation, security review, and testing before making adoption decisions.
+> [!NOTE]  
+> Each package is scored on their latest version. Scores are unlikely to change drasically between versions.
+
+#### CLI Usage
+
+```
+> cdk-construct-analyzer --help
+
+Usage: cdk-construct-analyzer <package> [options]
+
+Positionals:
+  package       NPM package name to analyze (Scored on the latest version)
+
+Options:
+ -v, --verbose  Show detailed signal information
```

Comment:
I suggest naming this `--detail`. `verbose` is commonly associated to logging level, not data.